app/scripts/components/datasets/s-explore/index.tsx

```diff
@@ -374,6 +385,33 @@ function DatasetsExplore() {
         return;
       }
 
+      if (currActiveData.basemapId) {
+        setBasemapId(currActiveData.basemapId);
+      } else {
+        setBasemapId(BASEMAP_ID_DEFAULT);
+      }
+    },
+    [activeLayer]
+  );
+
+  // On layer change, reset the basemapId.
```

Comment:
Is this comment at the right place? It seems to describe what https://github.com/NASA-IMPACT/veda-ui/pull/633/files#diff-be896ef6bddd109e493aede34acda96085cbbcfa648f310813b3f10842c2de61R388-R392 does. am I reading it wrong?

Reply:
good catch. had fixes this, but apparently not saved the file 😅